docs/concepts/glossary.md

```diff
@@ -34,7 +38,7 @@ A Blockchain is a growing list of records, known as blocks, that are linked usin
 
 ### Block
 
-A Block is a binary blob of data, identified by a [CID](#cid).
+A Block is a binary blob of data, identified by a [CID](#cid). I could be raw bytes or arbitrary data, or a chunk of serialized binary data encoded with [IPLD](#ipld) [codec](#codec).
```

Comment:
```suggestion
A Block is a binary blob of data identified by a [CID](#cid). It could be raw bytes of arbitrary data or a chunk of serialized binary data encoded with [IPLD](#ipld) [codec](#codec).
```